70.md

```diff
@@ -0,0 +1,51 @@
+NIP-70
+======
+
+Protected Events
+----------------
+
+`draft` `optional`
```

Comment:
```suggestion
`draft` `mandatory`
```

Reply:
I like the simplicity, but it's gotta be widely adopted for it to be useful.

Reply:
maybe the requirement that relays reject events with `"-"` tag by default should actually be added to NIP01, and then this NIP would define when events with `"-"` tag can be accepted (when the author matches authenticated user). that would suggest that rejection of events with `"-"` tag is expected behavior, while making it optional to accept them under very specific parameters.